app/models/institutions_archive.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require_dependency 'institution'
+
+class InstitutionsArchive < Institution
+  self.table_name = 'institutions_archives'
+
+  # class methods
+  # rubocop:disable Metrics/MethodLength
+  def self.archive_previous_versions(user)
+    production_version = Version.current_production.number
+    previous_version = Version.previous_production.number
+
+    begin
+      ActiveRecord::Base.transaction do
+        create_archives(previous_version, production_version)
+        Institution.where('version >= ? and version < ?', previous_version, production_version).delete_all
```

Comment:
Since `production_version` == `Version.production.newest.first` and `previous_version` == `Version.production.newest.second`, then the above lines will only ever archive/delete `previous_version`.
Thus `create_archives` only needs to take `previous_version` as a parameter (and alter method definition as appropriate).  Same with the deletion:
```suggestion
        Institution.where('version = ?', previous_version).delete_all
```